src/class_objectProcessorQueue.py

```diff
@@ -20,7 +20,7 @@ def put(self, item, block = True, timeout = None):
     def get(self, block = True, timeout = None):
         try:
             item = Queue.Queue.get(self, block, timeout)
-        except Queue.Empty as e:
+        except Queue.Empty:
             raise Queue.Empty()
```

Comment:
better to add logging here, because it will be easy to debug the application.

Reply:
I agree that this would be a good spot for logging However that is outside the scope of the work (fix codacy errors) that I was asked to do. Feel free to submit a separate PR.

Reply:
Logging not necessary here. This is a simple class extending Queue and this line is just a re-raising. It's up to wherever it's used to handle errors.

Reply:
Is that try/except needed here at all?

Reply:
@g1itch I think you're right. @coffeedogs If you can, please remove line 21, 23 and 24.

Reply:
Will do.